Package.swift

```diff
@@ -8,12 +8,12 @@ let package = Package(
     ],
     dependencies: [
         // 💧 A server-side Swift web framework.
-        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0"),
-        .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0"),
-        .package(url: "https://github.com/vapor/leaf.git", from: "4.0.0"),
-        .package(url: "https://github.com/vapor/fluent-postgres-driver.git", from: "2.0.0"),
+        .package(url: "https://github.com/vapor/vapor.git", from: "4.62.1"),
```

Comment:
Did we mean to bump all these? Thats a big 'ol jump!

Reply:
I'll remove the bumping of versions and pop those into a new PR, everything seems to work but we get more async functionality and especially things like security vulnerability fixes in the latest release (looking at the Vapor release notes anyways) so I think it's definitely needed. (https://github.com/vapor/vapor/releases)

Reply:
Removed updated versions in latest commit

Reply:
Oh fab, thank you for pulling out!

Sounds like the right play 👍🏻